src/main/resources/ink/abb/pogo/scraper/gui/index.html

```diff
@@ -10,7 +10,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Pokemon Go Bot GUI</title>
+    <title>{{username}} - Pokemon Go Bot GUI</title>
```

Comment:
Yes, it will not change but data should be added in the client and not Server (my opinion)